src/test/java/core/basesyntax/strategy/PurchaseOperationTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.StorageImpl;
+import core.basesyntax.dto.Transaction;
+import org.junit.jupiter.api.Test;
+
+import static core.basesyntax.model.OperationEnum.PURCHASE;
+import static org.junit.jupiter.api.Assertions.*;
+
+class PurchaseOperationTest {
+
+    private static final Storage storage = new StorageImpl();
+    private static final PurchaseOperation purchaseOperation = new PurchaseOperation();
```

Comment:
I think storage and purchaseOperation can be just instance variables, we don't need to access them in static methods:
```suggestion
    private final Storage storage = new StorageImpl();
    private final PurchaseOperation purchaseOperation = new PurchaseOperation();
```

Reply:
Thanks for your advice! Done.